plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -271,8 +254,280 @@ type Query {
 }
 `;
 
-// Pass implementation of MetronomeClient to the resolvers for easier testability
-export const resolvers = (): IResolvers => ({
+interface ResolverArgs {
+  fetchJobDetail: (id: string) => Observable<MetronomeClient.JobDetailResponse>;
+  pollingInterval: number;
+}
+
+function isActiveRun(arg: any): arg is MetronomeClient.ActiveJobRun {
+  return arg.jobId !== undefined;
+}
+
+const typeResolvers = {
```

Comment:
I would very much like seeing those as separate modules that export functions / consts